specification/azurelargeinstance/resource-manager/Microsoft.AzureLargeInstance/preview/2023-07-20-preview/examples/AzureLargeInstance_Get.json

```diff
@@ -0,0 +1,55 @@
+{
+  "parameters": {
+    "api-version": "2023-07-20-preview",
+    "subscriptionId": "f0f4887f-d13c-4943-a8ba-d7da28d2a3fd",
+    "resourceGroupName": "myResourceGroup",
+    "azureLargeInstanceName": "myAzureLargeInstance"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/f0f4887f-d13c-4943-a8ba-d7da28d2a3fd/resourceGroups/myResourceGroup/providers/Microsoft.AzureLargeInstance/AzureLargeInstances/myAzureLargeInstance",
+        "location": "westus2",
+        "name": "myAzureLargeInstance",
+        "tags": {
+          "key": "value"
+        },
+        "type": "Microsoft.AzureLargeInstance/AzureLargeInstance",
```

Comment:
>Microsoft.AzureLargeInstance [](http://example.com/codeflow?start=17&length=28)

The type in all examples would be plural. For example, Microsoft.AzureLargeInstances/AzureLargeInstance here.

Reply:
Good point! Thanks for catching those.